app-misc/anki/anki-23.12.1.ebuild

```diff
@@ -698,7 +698,7 @@ LICENSE="AGPL-3+ BSD public-domain"
 # Dependent crate licenses
 LICENSE+="
 	Apache-2.0 Apache-2.0-with-LLVM-exceptions BSD-2 BSD CC0-1.0 ISC MIT
-	MPL-2.0 Unicode-DFS-2016 ZLIB
+	MPL-2.0 openssl Unicode-DFS-2016 ZLIB
```

Comment:
That's not the correct approach, as `pycargoebuild` will overwrite this. You need to append it separately.